lib/quicksilver-config-client.js

```diff
@@ -51,6 +60,24 @@ class QuicksilverConfigClient {
   getExchangeSettings() {
     return true;
   }
+
+  /**
+   * Check that configuration path (qsConfigPath) value defined in .env as QS_CONFIG_PATH
+   * is a valid path.
+   */
+  configFileFound() {
+    return new Promise((resolve) => {
+      fs.lstat(this.qsConfigPath, (err, res) => {
```

Comment:
By convention, `response` should be renamed to `stats`.
See https://nodejs.org/api/fs.html#fs_fs_lstat_path_callback:
> The callback gets two arguments `(err, stats)` where `stats` is a `fs.Stats` object.